errors/errors.go

```diff
@@ -43,42 +48,51 @@ func (err *QueryError) Error() string {
 		return "<nil>"
 	}
 	str := fmt.Sprintf("graphql: %s", err.Message)
-	if err.Extensions.Code != 0 {
-		str += fmt.Sprintf(" code: %d", err.Extensions.Code)
+
+	if err.Extensions["Code"] != 0 && err.Extensions["Code"] != nil {
+		str += fmt.Sprintf(" code: %d", err.Extensions["Code"])
 	}
-	if err.Extensions.DeveloperMessage != "" {
-		str += fmt.Sprintf(" developerMessage: %s", err.Extensions.DeveloperMessage)
+	if err.Extensions["DeveloperMessage"] != "" && err.Extensions["Code"] != nil {
+		str += fmt.Sprintf(" developerMessage: %s", err.Extensions["DeveloperMessage"])
 	}
-	if err.Extensions.MoreInfo != "" {
-		str += fmt.Sprintf(" moreInfo: %s", err.Extensions.MoreInfo)
+	if err.Extensions["MoreInfo"] != "" && err.Extensions["Code"] != nil {
+		str += fmt.Sprintf(" moreInfo: %s", err.Extensions["MoreInfo"])
 	}
-	if err.Extensions.Timestamp != "" {
-		str += fmt.Sprintf(" timestamp: %s", err.Extensions.Timestamp)
+	if err.Extensions["Timestamp"] != "" && err.Extensions["Code"] != nil {
+		str += fmt.Sprintf(" timestamp: %s", err.Extensions["Timestamp"])
 	}
+
```

Comment:
updated code syntax as Extensions is update from a struct type to a map type field.